subxt/src/metadata/metadata_type.rs

```diff
@@ -273,9 +294,10 @@ impl PalletMetadata {
 /// Metadata for specific events.
 #[derive(Clone, Debug)]
 pub struct EventMetadata {
-    pallet: String,
+    pallet: Arc<str>,
```

Comment:
can you explain why this is changed to an `Arc`?

Reply:
Ah sure; Basically because the `pallet` name is duplicated across every event in a pallet ,so instead of `clone()`int a String to store with every event I just stuck it behind an arc so that it's an increment instead.

Probably worth a comment in the code :)

Reply:
(comment added)